model/src/test/kotlin/OrtResultTest.kt

```diff
@@ -127,6 +127,7 @@ class OrtResultTest : WordSpec({
     "getDefinitionFilePathRelativeToAnalyzerRoot()" should {
         "use the correct vcs" {
             val vcs = VcsInfo(type = VcsType.GIT, url = "https://example.com/git", revision = "")
+            val provenance = RepositoryProvenance(vcsInfo = vcs, resolvedRevision = HashAlgorithm.SHA1.emptyValue)
```

Comment:
Why is `resolvedRevision` set to `HashAlgorithm.SHA1.emptyValue` instead of `vcs.revision`? This seems to be a mismatch.

Reply:
Ok, to answer my own question, I guess this is because `resolvedRevision` must not be blank.

So I wonder whether we should make `VcsInfo.EMPTY.revision` equal to `HashAlgorithm.SHA1.emptyValue` instead... but I guess not, as using `HashAlgorithm.SHA1.emptyValue` for `Repository.EMPTY` was kind of a work-around to begin with.